Assets/SodiePopper.cs

```diff
@@ -96,7 +99,9 @@ private void FixedUpdate()
             if (accelerationDiff - playerDiff * playermovementRelativeAdjustment >= shakeCorrectionReloadThreshold && Time.fixedTime - lastReload > reload_delay)
             {
                 lastReload = Time.fixedTime;
-
+                if (gunController && gunController.stats.Magazine != gunController.stats.Ammo)
```

Comment:
Should check whether ammo < magazine, with a treshold for float inaccuracies. Seems to play the sound when it is full right now.

Reply:
Right into technical debt it goes 🤠